connect/runtime/src/test/java/org/apache/kafka/connect/util/clusters/EmbeddedConnectCluster.java

```diff
@@ -296,8 +419,14 @@ public Builder brokerProps(Properties brokerProps) {
             return this;
         }
 
+        public Builder failOnUngracefulShutdown(boolean fail) {
```

Comment:
We probably need JavaDoc for this method, because of the double-negative and because it's unclear what the alternative of not failing means. It's kind of clear looking at the code, but users will have less understanding.

Reply:
Refactored and removed double negative. Also added javadoc